include/aspect/compat.h

```diff
@@ -23,6 +23,26 @@
 
 #include <aspect/global.h>
 
+#if DEAL_II_VERSION_GTE(8,4,0)
+#else
+template <int dim>
+dealii::Tensor<1,dim> cross_product(const dealii::Tensor<1,dim> &a, const dealii::Tensor<1,dim> &b)
```

Comment:
I assume if I put them into dealii, that the compiler won't be able to figure out whether to call ```void cross_product(Tensor, Tensor)``` or ```Tensor cross_product(Tensor, Tensor)``` below (where I want the former). How would I get around that?